app/assets/javascripts/Components/Helpers/range_filter.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+
+export class RangeFilter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {showErrorMessage: false};
```

Comment:
Don't use state here; as we discussed at our meeting, it should be possible to conditionally display the div using CSS. Do that or ask additional questions.